web/src/message_list.js

```diff
@@ -46,10 +51,9 @@ export class MessageList {
             });
         }
 
-        // The table_name is the outer HTML element for this message
-        // list in the DOM.
-        const table_name = opts.table_name;
-        this.table_name = table_name;
+        // Attach message list to DOM.
+        $("#message-lists-container").append(render_message_list({message_list_id: this.id}));
+        this.$list = $(`.message-list[data-message-list-id="${CSS.escape(this.id)}"]`);
```

Comment:
I think these belong in `message_list_view.js`: we have no other DOM manipulation jQuery usage in this file.

Reply:
Sounds good, changed.